assets/agenda/components/ToBeConfirmedLabel.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import classNames from  'classnames';
+import {IAgendaItem, IListConfig, IPreviewConfig} from 'interfaces';
+import {isTopStory} from 'agenda/utils';
+import {gettext} from 'utils';
+
+interface IProps {
+    item: IAgendaItem;
+    size?: 'normal' | 'big';
+}
+
+export default function ToBeConfirmedLabel({item, size}: IProps) {
+    const classes = classNames('label label--rounded label--orange', {
```

Comment:
not sure if `label--orange` makes sense in regards to theming, `label--warning` might be better

Reply:
its not my class name... i just use the firtz'es one

Reply:
what do you think @fritzSF ?

Reply:
@petrjasek I agree with you, but this is how the class names for the labels are set up at the moment. It's nothing new, it is like that for the last six years. We should change to smenatic names at some point for sure.